tgui/packages/tgui/interfaces/BarberingMenu.js

```diff
@@ -0,0 +1,140 @@
+  const { act, data } = useBackend(context);
+  const { selected_hair_portion } = data;
+
+  return (
+    <Stack justify="space-between" fontSize="15px">
+      <Stack.Item>
+        <Button.Checkbox mx="20px" checked={selected_hair_portion === "bottom" ? 1 : 0} onClick={() => act("change_hair_portion", { "new_portion": "bottom" })}>Bottom Hair</Button.Checkbox>
+        <Button.Checkbox mx="20px" checked={selected_hair_portion === "middle" ? 1 : 0} onClick={() => act("change_hair_portion", { "new_portion": "middle" })}>Middle Hair</Button.Checkbox>
+        <Button.Checkbox mx="20px" checked={selected_hair_portion === "top" ? 1 : 0} onClick={() => act("change_hair_portion", { "new_portion": "top" })}>Top Hair</Button.Checkbox>
+      </Stack.Item>
+      <Stack.Item width="2px">
+        <Box backgroundColor="white" width="2px" height="100%" />
+      </Stack.Item>
+      <Stack.Item shrink>
+        <Button bold color="red" icon="cut" onClick={() => act("do_hair", { "style_id": null })}>Create Wig</Button>
+      </Stack.Item>
+    </Stack>
+  );
+};
+
+const HairPreview = (props, context) => {
+  const { act, data } = useBackend(context);
+  const { hair_style, hair_name } = props;
+  return (
+    <Section width="140px" direction="column" align="center">
+      <Stack inline justify="space-between">
+        <Stack.Item>
+          <Image pixelated width="60px" height="100px" src={`${hair_style["hair_icon"]}`} />
+          <Box width="100%" fontSize="15px" textAlign="center" mb="10px">{hair_name}</Box>
+          <Button color="blue" height="20px" icon="cut" onClick={() => act("do_hair", { "style_id": hair_style["hair_id"] })}>Cut</Button>
+          <Button color="blue" height="20px" icon="eye" onClick={() => act("update_preview", { "action": "new_hair", "style_id": hair_style["hair_id"] })}>Preview</Button>
+        </Stack.Item>
+      </Stack>
+    </Section>
+  );
+};
+
+const PreviewWindow = (props, context) => {
+  const { act, data } = useBackend(context);
+  const { preview, current_hair_style } = data;
+  return (
+    <Section>
+      <Stack justify="space-around" height="200px" wrap="wrap">
+        <Stack.Item width="100%">
+          <Button icon="rotate-left" color="red" height="22px" width="100%" onClick={() => act("update_preview", { "action": "reset" })}>Reset</Button>
+        </Stack.Item>
+        <Stack.Item>
+          <ByondUi
+            params={{
+              id: preview,
+              type: "map",
+            }}
+            style={{
+              width: "80px",
+              height: "160px",
+            }} />
+        </Stack.Item>
+      </Stack>
+      <hr />
+      <Stack wrap="wrap" align="center">
+        <Stack.Item>
+          <Box width="100%" bold>Top Hair: {current_hair_style["top"]}</Box>
+          <Box width="100%" bold>Middle Hair: {current_hair_style["middle"]}</Box>
+          <Box width="100%" bold>Bottom Hair: {current_hair_style["bottom"]}</Box>
+        </Stack.Item>
+      </Stack>
```

Comment:
```suggestion
      <LabeledList>
        <LabeledList.Item label="Top Hair">{current_hair_style["top"]}</LabeledList.Item>
        <LabeledList.Item label="Middle Hair">{current_hair_style["middle"]}</LabeledList.Item>
        <LabeledList.Item label="Bottom Hair">{current_hair_style["bottom"]}</LabeledList.Item>
      </LabeledList>
```
Untested, but this seems like a labeled list to me? https://github.com/goonstation/goonstation/blob/master/tgui/docs/component-reference.md#labeledlist

Reply:
it works nicely, i'll add it in the next commit